cli/main.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import typer
+from functools import reduce
+
+sys.path.append('../')
+from web.mappings import map 
```

Comment:
```suggestion
import sys

import typer
from functools import reduce

from web.mappings import map
sys.path.append('../')
```

So apparently python's style guide suggests imports to be sorted in the following way: 
https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

We don't need to worry too much about it though for this project.

Reply:
Also, it looks like with this change we can just run it directly using `python -m cli.main get-phrase 222426282`